query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -112,7 +111,13 @@ final case class LabelValuesExec(queryContext: QueryContext,
   def doExecute(source: ChunkSource,
                 querySession: QuerySession)
                (implicit sched: Scheduler): ExecResult = {
-    val parentSpan = Kamon.currentSpan()
+    if (!source.isReadyForQuery(dataset, shard)) {
+      if (!queryContext.plannerParams.partialResultsOk) {
+        throw new ServiceUnavailableException(s"Unable to answer query since shard $shard is still bootstrapping")
+      }
+      querySession.resultCouldBePartial = true
+      querySession.partialResultsReason = Some(s"Shard $shard still bootstrapping")
```

Comment:
Should this be list to avoid overriding when multiple shards are still bootstrapping ?

Reply:
Not needed, in favor of generic message.